### PR TITLE
Banner

### PR DESCRIPTION
**Pull request by vz3**
# Notes

This implements the notice/banner per #672

## Outstanding Issues

- [x] Keyboard doesn't fire the jQuery click to close the notice
- [x] **Help wanted**: The state of the toggle is not persistent as user navigates the site (i.e. it resets every page)

# Screen shots

## Notice active desktop
screen shot 2015-04-08 at 12 06 32 pm

## Notice closed desktop
screen shot 2015-04-08 at 12 06 38 pm

## Notice active mobile
screen shot 2015-04-08 at 12 06 50 pm

## Notice closed mobile
screen shot 2015-04-08 at 12 07 03 pm

### CODE REVIEW

**Comment by noahmanger** [comment]
Let me know what you decide regarding tracking the state of the banner. I'm implementing this now over at FEC.

**Comment by khandelwal** [comment]
With @theresaanna's new pull request, are some of the issues mentioned here resolved?

**Comment by theresaanna** [comment]
@khandelwal I just checked off the one resolved item.

**Comment by vz3** [comment]
Yes just the one

**Comment by vz3** [comment]
@theresaanna Do you have any idea why the close button doesn't get fired by tabbing over and hitting enter? cc @nickbristow

**Comment by theresaanna** [comment]
@vz3 I don't know why it doesn't work, but I have a pull request to make it work. https://github.com/18F/foia-hub/pull/727. I'm guessing the difference is image vs text.

cc: @nickbristow

**Comment by vz3** [comment]
Pulling out last issue for further discussion:
- [ ] Original comps in #647 show the "in development copy" disappearing by default. This isn't implemented. (For the record, I think it's confusing to show/hide inline content like this.)

File: foia_hub/templates/includes/nav.html [hunk 1]
```diff
@@ -7,7 +7,17 @@
     <div class="container">
       <div class="usa--content">
         <span class="official"><img alt="US flag signifying that this is a United States Federal Government website" src="{{ static("images/us_flag_small.png")}}" class="flag">&nbsp;&nbsp;An official website of the U.S. Government</span>
-        <span class="alpha">This website is in alpha. <a href="/about#feedback">Help us improve.</i></a></span>
+        <span class="alpha">This website is in develpment. <a id="notice--toggle" href="#">Learn more</a></span>
```
Comment (khandelwal) [inline foia_hub/templates/includes/nav.html]: s/develpment/development

**Pull request merged**